project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.7.2
+sbt.version=1.9.7
```

Comment:
Just realised, the readme still specifies 1.6.x
Maybe change in the readme to "the sbt version specified in project/build.properties" ?

Reply:
updated readme with sbt 1.9.7